curriculum/challenges/english/25-front-end-development/quiz-testing/66f1aeffc5774ac692112a7e.md

```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
-Placeholder distractor 2
+Unit testing
 
 ---
 
-Placeholder distractor 3
+Usability testing
 
 #### --answer--
 
-Placeholder answer
+Compatibility testing
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+In the context of web accessibility, what does the acronym WCAG stand for?
 
 #### --distractors--
 
-Placeholder distractor 1
+Web Compliance and Accessibility Guidelines
 
 ---
 
-Placeholder distractor 2
+World Computer Accessibility Group
 
 ---
 
-Placeholder distractor 3
+Web Content Accessibility Grading
 
 #### --answer--
 
-Placeholder answer
+Web Content Accessibility Guidelines
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following is NOT typically considered a part of shift-left testing practices?
 
 #### --distractors--
 
-Placeholder distractor 1
+Early integration of testing in the development process
 
 ---
 
-Placeholder distractor 2
+Increased automation of tests
 
 ---
 
-Placeholder distractor 3
+Involving testers from the beginning of the project
 
 #### --answer--
 
-Placeholder answer
+Performing all tests after the development phase
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the primary purpose of smoke testing in software development?
 
 #### --distractors--
 
-Placeholder distractor 1
+To test the system under extreme stress
 
 ---
 
-Placeholder distractor 2
+To check for security vulnerabilities
 
 ---
 
-Placeholder distractor 3
+To ensure all features are working perfectly
 
 #### --answer--
 
-Placeholder answer
+To verify that the most critical functions of a system are working
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which testing practice involves deliberately introducing faults into a system to test its fault-tolerance capabilities?
 
 #### --distractors--
 
-Placeholder distractor 1
+Regression testing
 
 ---
 
-Placeholder distractor 2
+Load testing
 
 ---
 
-Placeholder distractor 3
+Usability testing
 
 #### --answer--
 
-Placeholder answer
+Chaos testing
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What does TDD stand for in software development methodologies?
 
 #### --distractors--
 
-Placeholder distractor 1
+Test-Driven Deployment
 
 ---
 
-Placeholder distractor 2
+Total Design Development
 
 ---
 
-Placeholder distractor 3
+Technical Design Document
 
 #### --answer--
 
-Placeholder answer
+Test-Driven Development
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following is an example of a non-functional testing type?
 
 #### --distractors--
 
-Placeholder distractor 1
+Unit testing
 
 ---
 
-Placeholder distractor 2
+Integration testing
 
 ---
 
-Placeholder distractor 3
+User acceptance testing
 
 #### --answer--
 
-Placeholder answer
+Performance testing
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which external CI/CD tool is commonly used for automating builds, tests, and deployments?
 
 #### --distractors--
 
-Placeholder distractor 1
+Jenkins
```

Comment:
I've seen Jenkins in action and it does certainly does allow builds to be automated , tested, and deployed.

Reply:
Ok I will change it